pr/action.yml

```
@@ -328,3 +337,13 @@ runs:
         commit-message: ${{ steps.commit-message.outputs.value }}
         add-paths: .tool-versions
         signoff: ${{ inputs.signoff }}
+    - name: Auto-approve Pull Request
+      if: ${{ steps.update.outputs.updated-count != '0' &&
+        steps.pr-modified.outputs.value != 'true' &&
+        steps.pr-exists.outputs.value != 'true' &&
+        inputs.pr-auto-approve == 'true' }}
```

I'd prefer to use an [output of `peter-evans/create-pull-request`](https://github.com/peter-evans/create-pull-request/blob/84ae59a2cdc2258d6fa0732dd66352dddae2a412/README.md#action-outputs) here which _should_ be more explicit. I believe this should have the desired effect.

```suggestion
      if: ${{ steps.create-pr.outputs.pull-request-operation == 'created' &&
              inputs.pr-auto-approve == 'true' }}
```

That is, we only approve when the Pull Request is first created and the user configured it to do so.